stdlib/Printf/src/Printf.jl

```diff
@@ -1059,13 +1061,16 @@ function ini_dec(x::BigInt, n::Int, digits)
     end
     d = Base.ndigits0z(x)
     if d <= n
-        info = decode_dec(x)
+        info = decode_dec(x, digits)
         d == n && return info
```

Comment:
```suggestion
probably a good idea to use the same convention as elsewhere
```suggestion
        len,pt,neg = decode_dec(x, digits)
        d == n && return (len,pt,neg)
```